Dockerfile

```diff
@@ -14,12 +14,13 @@ ENTRYPOINT [ "/sbin/tini", "--" ]
 FROM base AS build-base
 WORKDIR /build
 COPY package.json package-lock.json prisma ./
-RUN npm ci --omit=dev --frozen-lockfile \
- && npx prisma generate
+RUN npm ci --include=dev --frozen-lockfile 
+RUN npx prisma generate
 
 FROM build-base AS build
 COPY --chown=node:node . ./
 RUN npm run build
+RUN npm prune --production
```

Comment:
Same as comment before. Is there any reason to have two separate docker layers for this? Can it be 

```Dockerfile
RUN npm run build \
 && npm prune --production
```